src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -379,7 +379,7 @@ GMAPS_API_KEY = '{{ GMAPS_API_KEY }}'
 MAPBOX_ACCESS_TOKEN = '{{ MAPBOX_ACCESS_TOKEN }}'
 FORMTRANSLATE_TIMEOUT = 5
 
-{% if OLD_BROKER_URL %}
```

Comment:
It's a little confusing but `OLD_BROKER_URL` is something that's constructed from `OLD_AMQP_HOST` and other variables here: https://github.com/dimagi/commcare-cloud/blob/cf6490753bddb07cfd0175e7ce6ee3ea73f11db4/src/commcare_cloud/ansible/roles/commcarehq/vars/main.yml#L6

Reply:
Added ~a week ago in https://github.com/dimagi/commcare-cloud/pull/4065